[CI] CommonAnalysisClientYamlTestSuiteIT test {yaml=analysis-common/40_token_filters/stemmer_override file access} failing

**Build Scans:**
- [elasticsearch-periodic #6237 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/ukgwfgphppna6)
- [elasticsearch-pull-request #55436 / part-1-entitlements](https://gradle-enterprise.elastic.co/s/cpfulq7iht4j6)
- [elasticsearch-periodic #6230 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/7conb7ov7yx6g)
- [elasticsearch-periodic #6223 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/3tjhlqybafahw)
- [elasticsearch-periodic #6216 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/ob3mfihx7224s)
- [elasticsearch-periodic #6209 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/ozizjblmu33yi)
- [elasticsearch-periodic #6202 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/rgdid7xy5jouq)
- [elasticsearch-periodic #6195 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/5sexz3zbpvf3i)
- [elasticsearch-periodic #6188 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/fcc44vfc4pe4m)
- [elasticsearch-periodic #6181 / openjdk24_checkrestcompat_java-matrix](https://gradle-enterprise.elastic.co/s/umrr5tovkw3dy)

**Reproduction Line:**
```
./gradlew ":modules:analysis-common:yamlRestCompatTest" --tests "org.elasticsearch.analysis.common.CommonAnalysisClientYamlTestSuiteIT.test {yaml=analysis-common/40_token_filters/stemmer_override file access}" -Dtests.seed=4CE3635447C07F64 -Dtests.locale=dav-Latn-KE -Dtests.timezone=Etc/GMT+7 -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.analysis.common.CommonAnalysisClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Byaml%3Danalysis-common%2F40_token_filters%2Fstemmer_override%20file%20access%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [analysis-common/40_token_filters:1132]: expected [400] status code but api [indices.create] returned [500 Internal Server Error] [{"error":{"root_cause":[{"type":"runtime_exception","reason":"Invalid Keyword override Rule:-XX:+UseG1GC","stack_trace":"org.elasticsearch.ElasticsearchException$1: Invalid Keyword override Rule:-XX:+UseG1GC\n\tat org.elasticsearch.server@9.1.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:725)\n\tat org.elasticsearch.server@9.1.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:639)\n\tat org.elasticsearch.server@9.1.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:202)\n\tat org.elasticsearch.server@9.1.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:161)\n\tat org.elasticsearch.server@9.1.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:122)\n\tat org
[truncated]
```

**Issue Reasons:**
- [main] 10 consecutive failures in step openjdk24_checkrestcompat_java-matrix
- [main] 11 failures in test test {yaml=analysis-common/40_token_filters/stemmer_override file access} (1.1% fail rate in 966 executions)
- [main] 10 failures in step openjdk24_checkrestcompat_java-matrix (100.0% fail rate in 10 executions)
- [main] 10 failures in pipeline elasticsearch-periodic (100.0% fail rate in 10 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -63,9 +63,6 @@ tests:
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=nodes.stats/11_indices_metrics/indices mappings exact count test for indices level}
   issue: https://github.com/elastic/elasticsearch/issues/120950
-- class: org.elasticsearch.analysis.common.CommonAnalysisClientYamlTestSuiteIT
-  method: test {yaml=analysis-common/40_token_filters/stemmer_override file access}
-  issue: https://github.com/elastic/elasticsearch/issues/121625
 - class: org.elasticsearch.test.rest.ClientYamlTestSuiteIT
   method: test {yaml=snapshot.delete/10_basic/Delete a snapshot asynchronously}
   issue: https://github.com/elastic/elasticsearch/issues/122102

